packages/office-ui-fabric-react/src/components/HoverCard/examples/HoverCard.Target.Example.tsx

```diff
@@ -46,8 +47,8 @@ class HoverCardField extends BaseComponent<IHoverCardFieldProps, IHoverCardField
           this.state.contentRendered &&
           <HoverCard
             expandingCardProps={ this.props.expandingCardProps }
-            instantOpenOnClick={ true }
             target={ this.state.contentRendered }
+            cardDismissDelay={ 300 }
```

Comment:
Just thinking out loud--from a guidance/usage perspective, it may be useful to explicitly call out in the example that you're adding a delay here.